WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -569,11 +568,32 @@ private extension DefaultProductFormTableViewModel {
             NSLocalizedString("Variations",
                               comment: "Title of the Product Variations row on Product main screen for a variable product")
 
+        static func variationsDetail(count: Int) -> String {
+            let format: String = {
+                switch count {
+                case 0:
+                    return ""
+                case 1:
+                    return NSLocalizedString("%1$ld variation",
+                                             comment: "Format for the variations detail row in singular form. Reads, `1 variation`")
+                default:
+                    return NSLocalizedString("%1$ld variations",
+                                             comment: "Format for the variations detail row in plural form. Reads, `2 variations`")
+                }
+            }()
+
+            return String.localizedStringWithFormat(format, count)
+        }
```

Comment:
This is a really clear way to handle the singular/plural cases! ⭐  I wish we could properly handle plurals for languages (like Russian) that have different plural forms (not just 1 or many), but until we can do that this is great. (We don't have an open issue for supporting different plural forms, but it's the same issue described in https://github.com/wordpress-mobile/WordPress-iOS/issues/6327.)

Reply:
[Stringdict](https://samwize.com/2018/07/19/how-stringsdict-and-nslocalizedstring-works/) would be the tool to help us with that but I think it is not supported by Glotpress yet :-(